### PR TITLE
K8s deployer

### PR DESCRIPTION
**Pull request by PiotrProkop**
Hi @alekssaul i prepared a K8s manifest for deploying stackanetes on kubernetes and fixed minor issues in code.

### CODE REVIEW

**Comment by PiotrProkop** [comment]
@ss7pro review please :smile:

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -42,10 +42,19 @@ echo Executing ansible playbook
 cd ansible
 ansible-playbook site.yml
 # there seems to be an extra horizon yml file that shouldn't get executed with "kolla-k8s run all"
```
Comment (DTadrzak) [inline docker-entrypoint.sh]: remove that comment too

File: stackanetes.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    app: stackanetes
+  name: stackanetes-deployer
+spec:
+  restartPolicy: OnFailure
+  containers:
+  - name: stackanetes-base
+    #insert a valid image name
+    image: stackanetes-deployer
+    env:
+    - name: dest_yml_files_dir
```
Comment (DTadrzak) [inline stackanetes.yml]: uppercase

File: stackanetes.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      value: "/var/lib/kolla-k8s"
+    - name: NEUTRON_HOSTIF
+      value: "eno2"
+    - name: KOLLA_K8s_YMLPATH
+      value: "/var/lib/kolla-k8s"
+    - name: host_interface
```
Comment (DTadrzak) [inline stackanetes.yml]: same_here

File: stackanetes.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  containers:
+  - name: stackanetes-base
+    #insert a valid image name
+    image: stackanetes-deployer
+    env:
+    - name: dest_yml_files_dir
+      value: "/var/lib/kolla-k8s"
+    - name: NEUTRON_HOSTIF
+      value: "eno2"
+    - name: KOLLA_K8s_YMLPATH
```
Comment (DTadrzak) [inline stackanetes.yml]: uppercase

File: stackanetes.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+spec:
+  restartPolicy: OnFailure
+  containers:
+  - name: stackanetes-base
+    #insert a valid image name
+    image: stackanetes-deployer
+    env:
+    - name: dest_yml_files_dir
+      value: "/var/lib/kolla-k8s"
+    - name: NEUTRON_HOSTIF
```
Comment (DTadrzak) [inline stackanetes.yml]: remove that variable, it's unnecessary.

File: stackanetes.yml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+metadata:
+  labels:
+    app: stackanetes
+  name: stackanetes-deployer
+spec:
+  restartPolicy: OnFailure
+  containers:
+  - name: stackanetes-base
+    #insert a valid image name
+    image: stackanetes-deployer
```
Comment (DTadrzak) [inline stackanetes.yml]: build image, push it on quay.io/stackanetes/stackanetes-deployer:0.0.1 and set it as a default value.

**Comment by DTadrzak** [comment]
lgtm, I think you should add information to README. Just simple description of this alternative stackanetes deployment, how does it work and how to run it.

File: stackanetes.yml (hunk 6 above)
Comment (PiotrProkop) [inline stackanetes.yml]: I will build image when this branch will be merged into master branch.

**Comment by ss7pro** [comment]
LGTM, would be nice to have some readme.

**Comment by philips** [comment]
@PiotrProkop @ss7pro shall we just merge this all to master?

**Comment by PiotrProkop** [comment]
@philips yeah i just have to add README section and change default values for environment files

**Comment by PiotrProkop** [comment]
Okay i updated README and fix some env variables.

**Pull request merged**